sklearn/cluster/_bicluster.py

```diff
@@ -20,10 +20,6 @@
 from ..utils.validation import assert_all_finite, check_array
 
 
-__all__ = ['SpectralCoclustering',
```

Comment:
This will technically mean that people who previously imported * from this module will now get check_array in their locals... Not sure that matters

Reply:
True. Can't we just keep this? What's the point of removing it?

Reply:
> What's the point of removing it?

It suggests that users might be allowed to import from this. And there's no reason to protect users that want to `import *` IMO.

But OK I'll put it back.